geoh5py/data/reference_value_map.py

```diff
@@ -42,7 +43,12 @@ def __len__(self) -> int:
         return len(self._map)
 
     def __call__(self) -> dict:
-        return dict(self._map)
+        try:
```

Comment:
raising an error to catch it is not the best practice.

Maybe we could just convert to "O".
Or check if string, then convert.